content/zh/docs/reference/kubectl/_index.md

```diff
@@ -1,4 +1,1078 @@
+`plugin`    | `kubectl plugin [flags] [options]` | 提供用于与插件交互的实用程序。
+`port-forward`    | `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N] [flags]` | 将一个或多个本地端口转发到一个 Pod。
+`proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | 运行访问 Kubernetes API 服务器的代理。
+`replace`        | `kubectl replace -f FILENAME` |  基于文件或标准输入替换资源。
+`rollout`    | `kubectl rollout SUBCOMMAND [options]` | 管理资源的上线。有效的资源类型包括：Deployment、 DaemonSet 和 StatefulSet。
+`run`        | <code>kubectl run NAME --image=image [--env="key=value"] [--port=port] [--dry-run=server &#124; client &#124; none] [--overrides=inline-json] [flags]</code> | 在集群上运行指定的镜像。
+<!--
+`scale`        | <code>kubectl scale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --replicas=COUNT [--resource-version=version] [--current-replicas=count] [flags]</code> | Update the size of the specified replication controller.
+`set`    | `kubectl set SUBCOMMAND [options]` | Configure application resources.
+`taint`    | `kubectl taint NODE NAME KEY_1=VAL_1:TAINT_EFFECT_1 ... KEY_N=VAL_N:TAINT_EFFECT_N [options]` | Update the taints on one or more nodes.
+`top`    | `kubectl top [flags] [options]` | Display Resource (CPU/Memory/Storage) usage.
+`uncordon`    | `kubectl uncordon NODE [options]` | Mark node as schedulable.
+`version`        | `kubectl version [--client] [flags]` | Display the Kubernetes version running on the client and server.
+`wait`    | <code>kubectl wait ([-f FILENAME] &#124; resource.group/resource.name &#124; resource.group [(-l label &#124; --all)]) [--for=delete&#124;--for condition=available] [options]</code> | Experimental: Wait for a specific condition on one or many resources.
+-->
+`scale`        | <code>kubectl scale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --replicas=COUNT [--resource-version=version] [--current-replicas=count] [flags]</code> | 更新指定副本控制器的大小。
+`set`    | `kubectl set SUBCOMMAND [options]` | 配置应用资源。
+`taint`    | `kubectl taint NODE NAME KEY_1=VAL_1:TAINT_EFFECT_1 ... KEY_N=VAL_N:TAINT_EFFECT_N [options]` | 更新一个或多个节点上的污点。
+`top`    | `kubectl top [flags] [options]` | 显示资源（CPU、内存、存储）的使用情况。
+`uncordon`    | `kubectl uncordon NODE [options]` | 将节点标记为可调度。
+`version`        | `kubectl version [--client] [flags]` | 显示运行在客户端和服务器上的 Kubernetes 版本。
+`wait`    | <code>kubectl wait ([-f FILENAME] &#124; resource.group/resource.name &#124; resource.group [(-l label &#124; --all)]) [--for=delete&#124;--for condition=available] [options]</code> | 实验特性：等待一种或多种资源的特定状况。
+
+<!--
+To learn more about command operations, see the [kubectl](/docs/reference/kubectl/kubectl/) reference documentation.
+-->
+了解更多有关命令操作的信息，
+请参阅 [kubectl](/zh/docs/reference/kubectl/kubectl/) 参考文档。
+
+<!--
+## Resource types
+-->
+## 资源类型
+
+<!--
+The following table includes a list of all the supported resource types and their abbreviated aliases:
+-->
+下表列出所有受支持的资源类型及其缩写别名:
```

Comment:
nit: The original content is ending with `.` instead of `:`
Maybe the Mandarin part can keep `：` as is, but it's better to change the English part to prevent next translator change it back unintentionally.

Reply:
We replace ':' by '：'。